src/pages/publication/FilesAndLicensePanel.tsx

```diff
@@ -40,6 +40,7 @@ const FilesAndLicensePanel: React.FC<FilesAndLicensePanelProps> = ({ goToNextTab
   const { t } = useTranslation('publication');
   const { values }: FormikProps<Publication> = useFormikContext();
   const [isLicenseModalOpen, setIsLicenseModalOpen] = useState(false);
+  const shouldAllowMultipleFiles = true;
```

Comment:
Trenger vi en const her når vi bare sender den inn som en prop?

Reply:
Jeg la det inn slik for bedre lesbarhet. Om vi bare sender inn en 'true' må vi inn i funksjonen for å lese hva den står for.

Reply:
her er den satt inni komponenten f.eks.